### DISCUSSION TITLE
Does Typhoon Skeleton break creating Modular Pages from Admin ?

### DISCUSSION THREAD

**Discussion by dryland404** [getgrav/grav-premium-issues#345] [General]
Newish to Grav . 
Bought Typhoon to get a head start on a project . 
Making A page of modules seems broken. I'm follow examples from Youtube but it all looks different. 
https://www.youtube.com/watch?v=2KLHQKxWIRE

I add modules with sample content and nothing shows up on the page when testing a view of it .. 
Should I get a refund ?

**Comment by rhukster** [comment]
The youtube video you are following is for Antimatter theme, Typhoon as it's own modules that are not going to be compatible with Antimatter.

First you should read the Typhoon docs here: https://getgrav.org/premium/typhoon/docs

There's a section on this page about the Skeleton packages: https://getgrav.org/premium/typhoon/docs#skeleton-packages

These provide a pre-configured setup with sample pages/config/etc.  You should probably download the "One-Page" skeleton, and follow the instructions: https://github.com/trilbymedia/grav-skeleton-typhoon-onepage

Let me know if you continue to have problems...
